distributed/stealing.py

```diff
@@ -278,8 +282,11 @@ def move_task_request(
                 thief.occupancy,
             )
 
-            victim_duration = victim.processing[ts]
-
+            # TODO: occupancy no longer concats linearily so we can't easily
+            # assume that the network cost would go down by that much
```

Comment:
in different terms: "Occupancy by task" is no longer constant and we'd need to recompute it if this should be used for any decision.